src/node_nanomsg.cc

```diff
@@ -191,35 +190,40 @@ NAN_METHOD(Err) {
   info.GetReturnValue().Set(Nan::New(nn_strerror(nn_errno())).ToLocalChecked());
 }
 
-typedef struct nanomsg_socket_s {
-  uv_poll_t poll_handle;
-  uv_os_sock_t sockfd;
-  Nan::Callback *callback;
-} nanomsg_socket_t;
+class NanoMsgPollCtx {
+  private:
+  public:
+    uv_poll_t poll_handle;
+    uv_os_sock_t sockfd;
+    Nan::Callback *callback;
+    NanoMsgPollCtx (int s, bool is_sender, Local<v8::Function> cb) {
+      callback = new Nan::Callback(cb);
+      size_t siz = sizeof(uv_os_sock_t);
```

Comment:
Now I realized you may have misunderstood what I was trying to say:
I claim that `sizeof ( uv_os_sock_t)` is better than `sizeof( uv_os_sock_t)`, since the rest of the code follows the convention of keywords having a space before their parameter list, e.g. `if (foo)` vs `if(foo)`.